lib/pfconfig/namespaces/config/Pf.pm

```diff
@@ -177,6 +180,11 @@ sub build_child {
         port     => $webservices->{'port'},
     };
 
+
+    if (isenabled($Config{'captive_portal'}{'secure_redirect'}) && selfSigned()) {
+        $Config{'captive_portal'}{'secure_redirect'} = 'disabled';
```

Comment:
Add a logging statement stating the secure redirect is being ignored since the cert is self-signed.
I would go with info